themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -180,6 +181,7 @@ VuFind.register('itemStatuses', function ItemStatuses() {
       return;
     }
     if ($item.find('.hiddenId').length === 0) {
+      VuFind.emit("item-status-done");
```

Comment:
Do we want to mark the whole process done just because a single item was missing an ID?